### PR TITLE
Add a note to Security Authorization doc how to disable the permissions

### PR DESCRIPTION
**Pull request by sberyozkin**

### CODE REVIEW

File: docs/src/main/asciidoc/security-authorization.adoc [hunk 1]
```diff
@@ -152,6 +152,20 @@ That is if the security annotations do not define the access control. Defaults t
 and JAX-RS endpoints that do not have security annotations but are defined in classes that contain methods with
 security annotations. Defaults to `false`.
 
+=== Disabling the permissions
```
Comment (gastaldi) [inline docs/src/main/asciidoc/security-authorization.adoc:155]:
```suggestion
=== Disabling permissions 
```

File: docs/src/main/asciidoc/security-authorization.adoc [hunk 2]
```diff
@@ -152,6 +152,20 @@ That is if the security annotations do not define the access control. Defaults t
 and JAX-RS endpoints that do not have security annotations but are defined in classes that contain methods with
 security annotations. Defaults to `false`.
 
+=== Disabling the permissions 
+
+The permissions can be disabled at build time with a permissions policy specific `enabled` property, for example:
```
Comment (gastaldi) [inline docs/src/main/asciidoc/security-authorization.adoc:157]:
```suggestion
Permissions can be disabled at build time with an `enabled` property for each declared permission, for example: 
```

**Review by gastaldi — COMMENTED** [review]
Looks good, reworded a bit, see if that makes sense

**Comment by sberyozkin** [comment]
@gastaldi thanks for those improvements

**Review by gsmet — APPROVED** [review]

**Pull request merged**